### PR TITLE
Webapp: Update @reach-ui and react-focus packages versions

### PR DESCRIPTION
**Pull request by vovakulikov**
## Context

This PR simply updates some package versions and also brings a new tool for bundle debugging. Why is this important? Well, this versions updating reduces bundle size by 90kb in parsed size. Which is not so small a reduction. It isn't a lot by 90kb is 90kb. 

A reason we had an extra 90kb in our main initial bundle. Well prior to this PR we had different versions of reach-ui components. Their versions weren't synced with each other. And all of them had different versions of `reach-ui/utils` packages. Because of that, we had 6 copies of `reach-ui/utils` package.

The initial bundle after version updating is presented below. As you can see we still have duplicates in the bundle but they will be resolved in separate PRs later (I also don't necessarily have a clear understanding of how to remove these duplicates in the right way).

<table>
<tr><th>The main</th><th>With updated package versions</th></tr>
<tr><td>
Screenshot 2022-02-10 at 01 01 22

</td><td>

Screenshot 2022-02-08 at 19 58 53</td></tr>
</table>

### CODE REVIEW

File: client/shared/src/components/activation/__snapshots__/ActivationChecklist.test.tsx.snap [hunk 1]
```diff
@@ -111,11 +111,12 @@ exports[`ActivationChecklist render 0/1 complete 2`] = `
...
       >
         <button
-          aria-controls="panel--4--0"
+          aria-controls="panel--5--0"
           aria-expanded="false"
           class="btn btnLink list-group-item list-group-item-action button"
           data-reach-accordion-button=""
-          id="button--4--0"
+          data-state="collapsed"
+          id="button--5--0"
```
Comment (unclejustin) [inline client/shared/src/components/activation/__snapshots__/ActivationChecklist.test.tsx.snap:119]: Curious, why did these have to change?

Comment (vovakulikov) [reply client/shared/src/components/activation/__snapshots__/ActivationChecklist.test.tsx.snap:119]: @unclejustin I think the inner implementation of reach UI accordion and other packages that have been updated and the reach accordion packages depends on

**Review by valerybugakov — COMMENTED** [review]
> how to remove these duplicates in the right way

One way of doing it is to rely on the `resolutions` field in the `package.json`. Could it be the solution here?

File: client/shared/src/components/activation/ActivationChecklist.test.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ describe('ActivationChecklist', () => {
             expect(component.asFragment()).toMatchSnapshot()
         }
     })
-    test('render 1/1 complete', () => {
+    test.skip('render 1/1 complete', () => {
```
[the remark this answers is not in the document]
Comment (vovakulikov) [reply client/shared/src/components/activation/ActivationChecklist.test.tsx:47]: @valerybugakov Good catch I forgot to leave a comment there. After @reach accordion package update, it became a flaky snapshot test. An interesting part is that if you just run this snapshot alone it's fine but if you run three snapshots in this file in a row it starts failing

**Pull request merged**